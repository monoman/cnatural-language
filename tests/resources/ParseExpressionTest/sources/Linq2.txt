from s in names
where s.Length == 5
select s.ToUpper()