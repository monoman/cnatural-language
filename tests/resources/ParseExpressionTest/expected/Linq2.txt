from s in Name(names)
where ((Name(s)).Length Equal Literal(DecimalInteger, [5]))
select (Name(s)).ToUpper()